model/project.go

```diff
@@ -37,9 +37,10 @@ const (
 )
 
 type GetProjectTasksOpts struct {
-	Limit        int    `json:"num_versions"`
-	BuildVariant string `json:"build_variant"`
-	StartAt      int    `json:"start_at"`
+	Limit        int      `json:"num_versions"`
+	BuildVariant string   `json:"build_variant"`
+	StartAt      int      `json:"start_at"`
+	Requesters   []string `json:"requesters"`
```

Comment:
The parse function for this route should validate that if this is specified the requesters are valid.

Reply:
haha i assumed it would be in your pr but it depends on who can merge first now
added

Reply:
oh LOL didn't realize this was the same route haha

Reply:
Update my PR only actually adds this for the _new_ route, whereas you're modifying an existing one